features/mbedtls/importer/Makefile

```diff
@@ -27,20 +27,39 @@
 #
 
 # Set the mbed TLS release to import (this can/should be edited before import)
-MBED_TLS_RELEASE ?= mbedtls-2.13.1
+MBED_TLS_RELEASE ?= mbedtls-2.15.0
 
 # Translate between mbed TLS namespace and mbed namespace
 TARGET_PREFIX:=../
+TARGET_PREFIX_CRYPTO:=../mbed-crypto/
 TARGET_SRC:=$(TARGET_PREFIX)src
 TARGET_INC:=$(TARGET_PREFIX)inc
 TARGET_TESTS:=$(TARGET_PREFIX)TESTS
 
+# New folder structure is introduced here for targets with Secured-Partition-Environment
```

Comment:
What about targets that don't have an SPM? Won't they be missing some files?

Reply:
Jaeden points me to https://github.com/ARMmbed/Mbed-PSA-Services-Docs/blob/master/build-system/rfc.md which says that those targets will get the files via `PSA_SRV_IMPL`. Looks ok then. CI should catch a missing file anyway.

Reply:
It's complicated how files are includes on targets with or without SPM. See https://github.com/ARMmbed/Mbed-PSA-Services-Docs/blob/master/build-system/rfc.md for details on the labeling and component system used.